functions.php

```diff
@@ -260,4 +260,13 @@ function getScoreboard($DB) {
     return $out;
 }
 
+function getMentors($users) {
+    global $bot_token;
+    $result = json_decode(get_query_slack("https://slack.com/api/usergroups.users.list",array("token"=>$bot_token,"users"=>$users)),true);
```

Comment:
According to [the api method specification](https://api.slack.com/methods/usergroups.users.list), the two parameters the api needs are `token` and `usergroup`, not `token` and `users`